client/Makefile

```diff
@@ -1,5 +1,7 @@
 include ../includes.mk
 
+.PHONY: build
```

Comment:
not sure if there's something wrong with my environment but `make build` wouldn't work without that